yapio.c

```diff
@@ -2135,6 +2180,37 @@ yapio_display_result(const yapio_test_ctx_t *ytc)
     }
 }
 
+/* dump yapioMyRank's metadata array into a file */
+static void
+yapio_store_md_final_state()
+{
+    /* each rank writes its own md file '{filename}-md-{rank}' */
+    char md_file[PATH_MAX + 10];
+    snprintf(md_file, PATH_MAX, "%s-md-%d", yapioTestFileName, yapioMyRank);
```

Comment:
you should check for the return to see if you truncate the name

Reply:
adding check for return code != 0